Update torque sensor

ibaiape opened the pull request:
Contemplating up to 3 axis as 6 would imply a force sensor, communicated separately.

Measurements of a single axis, multi-axis sensors could publish each value in a separated named topic.

ahcorde commented:
@ibaiape can you fill all the values in the specs message with [this datasheet](https://www.me-systeme.de/product-pdf?product_id=1912&lang=en), which is the current torque torque sensor integrated in H-ROS.

ibaiape commented:
**max_torque**:  50

**resolution**: 0.1

**device_type**: 0

**force_direction**: 2

**max_rpm**: 0  # Not taken into account if the sensor type is static

**safe_overload**: 150
**max_overload**:  400

**axis**:  1

**max_rate**: ? # Unspecified in datasheet, assumed max possible physical rate, calculable with connector&cable max rate divided by measurement bit size

**min_ideal_temp**: -10
**max_ideal_temp**: 60

**min_operating_temp**:  -10
**max_operating_temp**:  85

**min_error_margin**: 0.1
**max_error_margin**: 0.1


Removed min_torque (as it's either 0 or negative max_torque).
Renamed "calibrated" temp to "ideal" temp for easier understanding.
Added resolution.
Renamed max/min "accuracy" for max/min "error margin".
Removed force_sensor, if there's a force reading it will be visible through it's own topics.
